week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -26,7 +43,9 @@ The function must:
 */
 
 function remove(arr, index) {
-  return; // complete this statement
+  modifiedArr = arr;
+  modifiedArr.splice(index, 1);
```

Comment:
You need to return the array with the elements before the removed element, concatenated with the elements after the removed element.